stock_immediate_transfer_disable/wizard/stock_immediate_transfer.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class StockImmediateTransfer(models.TransientModel):
+
+    _inherit = 'stock.immediate.transfer'
+
+    allow_imediate_transfer = fields.Boolean(
+        related='pick_id.picking_type_id.allow_imediate_transfer')
```

Comment:
@dufresnedavid It's better to explicit the readonly option here. Otherwise it will rewrite the allow_immediate_transfer value every time you confirm the wizard.

Reply:
ok